addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/ExtensionScriptsUI.java

```diff
@@ -246,6 +246,15 @@ public void postInstall() {
                 }
             }
         }
+
+        // Register outputs for all scripts in the tree model
+        getExtScript().getScriptTypes().stream()
```

Comment:
Should we do this before the above which might cause output?

Reply:
Good catch, tweaked